### ISSUE TITLE
mercurial fails to install with Python 3 on Windows

### ISSUE THREAD

**Issue by lazka** [python/pyperformance#66]
```
ERROR: Package 'mercurial' requires a different Python: 3.8.0 not in '~= 2.7'
```

See https://www.mercurial-scm.org/repo/hg/file/5.0.2/setup.py#l10

HGALLOWPYTHON3 needs to be set as well so it can be installed with setuptools.

**Comment by lazka** [comment]
I've opened a PR for this -> #67

**Comment by vstinner** [comment]
I disabled again hg_startup on Python 3, I closed your PR #67 (see the rationale there).
